_includes/header.html

```diff
@@ -5,7 +5,7 @@
       <div class="small-10 columns">
         <ul class="c-header-bar-items">
           <li class="c-header-bar__item c-header-bar__item--smallerSpace">
-	   Piráti - Pardubický kraj</li>
+	   <a href="/">Piráti - Pardubický kraj</a></li>
```

Comment:
Co tam dat liquid, at to je mene vazane na zmenu kodu.  [Viz](https://jekyllrb.com/docs/variables/) `site.url`

Reply:
uvodni stranka nezavisi na zmenach kodu.
site.url obsahuje absolutni adresu vcetne protokolu a fqdn - nic horsiho tam nacpat nemuzes

Reply:
pravda... {{ relative_url }}?